### DISCUSSION TITLE
Should we include Cashier translations?

### DISCUSSION THREAD

**Discussion by luisprmat** [Laravel-Lang/common#50] [Q&A]
Looking for translations in the entire Laravel ecosystem I find that there are a few missing tags that correspond some to paid products like Laravel Nova (paid) and others to [Laravel Cashier](https://laravel.com/docs/8.x/billing) (to add payments and subscriptions). There are very few translations (I found about 15 tags) for this free package.
Taking into account that it is an open-source product like the others that we support in this package, do you think we should add them? 
These translations are in the following files:
- [`resources/views/payment.blade.php`](https://github.com/laravel/cashier-stripe/blob/12.x/resources/views/payment.blade.php)
- [`src/Notifications/ConfirmPayment.php`](https://github.com/laravel/cashier-stripe/blob/12.x/src/Notifications/ConfirmPayment.php)

**Comment by luisprmat** [comment]
I have a PR ready for this, if it is decided that I upload it.

**Comment by masterix21** [comment]
I think we can.

**Comment by caouecs** [comment]
I agree

**Comment by andrey-helldar** [comment]
Laravel Cashier definitely needs to be added (at the time of this writing, the changes have already been made to several languages).

I also suggest adding support for Nova, since this is also a Laravel product, albeit a paid one. Does anyone have any translation keys used in Nova?

**Reply by masterix21** [reply]
Nova translation keys:
[lang.zip](https://github.com/Laravel-Lang/lang/files/6129777/lang.zip)

**Reply by andrey-helldar** [reply]
😀

![image](https://user-images.githubusercontent.com/10347617/110962569-2c685200-8362-11eb-848b-23205be24e4e.png)

**Reply by andrey-helldar** [reply]
I created a PR https://github.com/Laravel-Lang/lang/pull/1554

**Reply by andrey-helldar** [reply]
@Laravel-Lang/laravel-lang, see, guys
